src/lib.rs

```diff
@@ -11,6 +11,13 @@
 #![forbid(unsafe_code)]
 #![deny(missing_docs)]
 
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
+use std::fmt;
+#[cfg(not(feature = "std"))]
+use core::fmt;
```

Comment:
Is there a reason to use `std::fmt`? Can't we just always use `core::fmt`?

Reply:
As far as I understood, there isn't much difference between `std::fmt` and `core::fmt`. The former re-exports symbols from the latter.

Will change it to always `core::fmt` if that's preferred.